dotnet/samples/NCalcPlugins/LanguageCalculatorPlugin.cs

```diff
@@ -71,9 +73,15 @@ public LanguageCalculatorPlugin(IKernel kernel)
             skillName: nameof(LanguageCalculatorPlugin),
             functionName: "TranslateMathProblem",
             description: "Used by 'Calculator' function.",
-            maxTokens: 256,
-            temperature: 0.0,
-            topP: 1);
+            requestSettings: new AIRequestSettings()
+            {
+                ExtensionData = new Dictionary<string, object>()
+                {
+                    { "MaxTokens", 256 },
+                    { "Temperature", 0.0 },
+                    { "TopP", 1 },
+                }
+            });
```

Comment:
We don't know in advance which LLM will be used to run this plugin, which means that some of the hardcoded parameters might not be relevant to that particular LLM, or additional ones specific to ther LLM might be required.

Can we extend the `LanguageCalculatorPlugin` constructor with the `AIRequestSettings` parameter, allowing the plugin consumers to provide settings specific to LLMs the plugin runs against?

Reply:
We currently have hardcoded OpenAI specific request settings in multiple places in the code and this issue still remains to be resolved. This PR does not provide a solution to this problem. I have a task in the back log to address this issue.